src/swaps/hop/HopSwapProvider.ts

```diff
@@ -195,11 +195,26 @@ class HopSwapProvider extends SwapProvider {
     }
   }
 
+  _formatFee(fee: any, networkName: string, type: string ){
+    if(fee.maxFeePerGas && fee.maxPriorityFeePerGas){
```

Comment:
If you have type, you can discriminate the type ( i think)

Reply:
This type I am using for  "approve" or "send" it is different. 

I am not sure if the comment is related to that

Reply:
Fixed, thanks for the help!